src/qt/rpcconsole.cpp

```diff
@@ -295,6 +296,14 @@ void RPCConsole::setClientModel(ClientModel* model)
     }
 }
 
+void RPCConsole::setWalletModel(WalletModel* model)
+{
+    walletModel = model;
+    if (walletModel) {
+        ui->wallet_path->setText(walletModel->getWalletPath());
+    }
+}
```

Comment:
This is never called, so the label stays "N/A".
We should set the wallet model after constructing the RPCConsole object in `SettingsInformationWidget::openNetworkMonitor`.

Reply:
yeah, good catch.